runner/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM ubuntu:18.04
 
 ARG TARGETPLATFORM
-ARG RUNNER_VERSION=2.274.2
```

Comment:
Thanks for the fix! But could we keep this build-arg as this is required to inject the runner version from within our github actions workflow?

Reply:
> could be nice to build this daily so no need to keep bumping the version

Sorry I missed your comment. I got your intention, but doing so would prevent us from tagging container images with their RUNNER_VERSION.

I'd certainly investigate a way to automate it, probably reusing your snippet for getting the latestest available runner version. But it should be done in a separate PR!